spec/controllers/coronavirus_form/product_details_controller_spec.rb

```diff
@@ -100,9 +100,9 @@
   describe "POST submit" do
     before :each do
       session[session_key] = [{
-        "product_id" => product_id,
-        "medical_equipment_type" => I18n.t(
-          "coronavirus_form.medical_equipment_type.options.number_ppe.label",
+        product_id: product_id,
+        medical_equipment_type: I18n.t(
+          "coronavirus_form.questions.medical_equipment_type.options.number_testing_equipment.label",
```

Comment:
I'm surprised the tests weren't bombing because this lookup failed.

Reply:
We were updating the session store before doing any validation, so the test for updating the session variables always passed.